PyInstaller/compat.py

```diff
@@ -889,3 +891,30 @@ def check_requirements():
             raise SystemExit('PyInstaller cannot check for assembly dependencies.\n'
                              'Please install PyWin32.\n\n'
                              'pip install pypiwin32\n')
+
+
+@contextmanager
+def open_pysrc(filename):
```

Comment:
This is a good idea. I suggest moving this into modulgraph, where is similar code which could be updated.